home-application/pom.xml

```diff
@@ -84,6 +85,16 @@
             <artifactId>quartz-actuator</artifactId>
             <version>${quartz-actuator.version}</version>
         </dependency>
+        <dependency>
```

Comment:
It would be nice to add a description to the dependencies, indicating for what purpose they were added. E.g. <!-- description in start --> <!-- /same description in end -->

Reply:
It provides further annotations to method parameters to pull information out of the request. All the @*Param annotations take a key of some form which is used to look up the value required.